evap/evaluation/management/commands/send_reminders.py

```diff
@@ -50,7 +50,10 @@ def send_student_reminders():
 
         recipients = set()
         for evaluation in Evaluation.objects.filter(
-            state=Evaluation.State.IN_EVALUATION, vote_end_date__in=check_dates
+            state=Evaluation.State.IN_EVALUATION,
+            vote_end_date__in=check_dates,
+            vote_start_datetime__date__lt=datetime.date.today()
```

Comment:
In addition to what Richard said regarding time shenanigans, can you make sure that we call `date.today()` only once in the function? Otherwise we might get different dates back when sending reminders at 23:59:59